app/controllers/course/announcements_controller.rb

```diff
@@ -0,0 +1,45 @@
+class Course::AnnouncementsController < Course::ModuleController
```

Comment:
Is it here ? https://github.com/coursemology-collab/coursemology2/compare/lowjoel/announcements-module#diff-b8105aaef0213a9a65866e469fe32099R1